dashboard/app/views/levels/editors/_gamelab_jr.html.haml

```diff
@@ -0,0 +1,30 @@
+.field
+  = f.label :custom_blocks, 'Custom block defintions'
+  %p
+    Add some custom JSON block definitions
+    <a href="#" onClick="toggleBlock('custom_blocks_more'); return false">Read more...</a>
+  #custom_blocks_more{style: 'display: none'}
+    %p
+      Here is an example.
+    %pre
+      :preserve
+        {
```

Comment:
It looks like the field is expecting an array of objects.

Reply:
I also made it automatically wrap an object in an array if you have one block that you forgot to put into an array.